lib/codecov_opentelem/coverage_exporter.rb

```diff
@@ -78,14 +79,17 @@ def export(spans, timeout)
         span_hash[k] = v
       end
 
+      cov_string = cov.map{ |k, v| "\"#{k}\":[#{v.map{ |i| i.nil? ? "null" : i}.join(',')}]"}.join(',')
+      cov_string = "{\"coverage\":{#{cov_string}}}"
```

Comment:
```suggestion
      cov_string = JSON.generate({ 'coverage': cov })
```

[JSON#generate]

[JSON#generate]:
https://ruby-doc.org/stdlib-3.1.2/libdoc/json/rdoc/JSON.html#method-i-generate

Reply:
@mitchell-codecov will do this in the following PR